docs/spectrum/index.rst

```diff
@@ -10,110 +10,116 @@ Spectrum estimation and modeling (`gammapy.spectrum`)
 
 Introduction
 ============
+`gammapy.spectrum` holds functions and classes related to 1D region based spectral analysis.
+The basic of this type of analysis are explained in `this <https://github.com/gammapy/PyGamma15/tree/gh-pages/talks/analysis-classical>`__ talk
 
-`gammapy.spectrum` holds functions and classes to fit spectral models and compute flux points.
-
-Physical radiative models (synchrotron, inverse Compton and pion-decay emission)
-for arbitrary cosmic ray particle spectra are available in the `naima`_ package.
-
-Explain spectrum estimation basics.
-
-Define vocabulary.
+TODO: explain basics
 
 A good reference for the forward-folding on-off likelihood fitting methods is Section 7.5 "Spectra and Light Curves" in [Naurois2012]_,
 in publications usually the reference [Piron2001]_ is used.
 A standard reference for the unfolding method is [Albert2007]_.
 
-.. _spectrum_getting_started:
 
-Getting Started
-===============
+Spectral Fitting
+================
 
-Spectral fitting within Gammapy is most easily performed with the ``gammapy-spectrum`` command line tool. 
+.. _spectrum_command_line_tool:
 
-The spectral fitting command-line tool makes use of the data management functionality in Gammapy. In order to download an example dataset from the `gammapy-extra <https://github.com/gammapy/gammapy-extra>`__ repository and set up an example `gammapy.obs.DataManager` please follow the instructions in :ref:`obs_dm`. The following step assume you have this example data set. If you already have a data set, please modify the steps below accordingly.
+Command line tool
+-----------------
+
+Spectral fitting within Gammapy is most easily performed with the ``gammapy-spectrum`` command line tool.
+The example below shows how to use ``gammapy-spectrum`` by specifying analysis
+options in a YAML config file. It assumes you have the `gammapy-extra <https://github.com/gammapy/gammapy-extra>`__
```

Comment:
I'm trying to move all (or at least most) URLs to this file to make text in docs shorter / maintenance of URLs simpler:
https://github.com/gammapy/gammapy/blob/master/docs/references.txt

Can you please add this line to `references.txt`:
```
.. _gammapy-extra: https://github.com/gammapy/gammapy-extra
```
and then just use this here to create the reference:
```
`gammapy-extra`_
```
?